src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdater.java

```diff
@@ -71,8 +75,12 @@ public void processDailyList(DailyListEntity dailyListEntity) throws JsonProcess
                             courtHouseName, sitting.getCourtRoomNumber(),
                             caseNumber, dailyListHearing.getHearingDetails().getHearingDate()
                         );
+                        hearing.setCreatedBy(systemUserHelper.getSystemUser());
+                        hearing.setLastModifiedBy(systemUserHelper.getSystemUser());
```

Comment:
so we need to keep calling the method, or should we just call systemUserHelper.getSystemUser() once and reuse the result?

Reply:
ACCEPT.